scripts/bert/bertpass_gpu.cc

```diff
@@ -0,0 +1,450 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2019 by Contributors
+ * \file subgraph_lib.cc
+ * \brief subgraph operator implementation library file
+ */
+
+#include <math.h>
+#include <iostream>
+#include <algorithm>
+#include <unordered_set>
+#include <functional>
+#include "lib_api.h"
```

Comment:
```suggestion
#include "mxnet/lib_api.h"
```

Reply:
thanks 
I have tried, and also tried to print the file. It seems it is not there:
`TypeError: invalid file: PosixPath('/var/lib/jenkins/workspace/gluon-nlp-gpu-py3-master/conda/gpu/py3-master/lib/python3.5/site-packages/mxnet/include/mxnet/lib_api.h')`
but in the wheel it was included

Reply:
@MoisesHer I verified that installing https://repo.mxnet.io/dist/python/cu100/mxnet_cu100-1.7.0b20200809-py2.py3-none-manylinux2014_x86_64.whl locally does place the file at `/home/ubuntu/.pyenv/versions/3.8.3/lib/python3.8/site-packages/mxnet/include/mxnet/lib_api.h`.

You could print the output of `find /var/lib/jenkins -name lib_api.h` in a debug statement to find the file location.

Reply:
I was able to print the header file `/var/lib/jenkins/workspace/gluon-nlp-gpu-py3/conda/gpu/py3/lib/python3.5/site-packages/mxnet/include/mxnet/lib_api.h` within the CI (the previous error was related to python version, so actually the file existed)
It contains this version:
[https://github.com/apache/incubator-mxnet/blob/v1.6.x/include/mxnet/lib_api.h](https://github.com/apache/incubator-mxnet/blob/v1.6.x/include/mxnet/lib_api.h)
which does not correspond to MXNet 1.7x version:
[https://github.com/apache/incubator-mxnet/blob/v1.7.x/include/mxnet/lib_api.h](https://github.com/apache/incubator-mxnet/blob/v1.7.x/include/mxnet/lib_api.h)

Reply:
CI "stable version tests" tests that the branch works with MXNet 1.6:

https://github.com/dmlc/gluon-nlp/blob/v0.x/env/gpu/py3.yml#L36

If you don't support 1.6, you need to skip the test for the "stable version tests". However, given that the 1.7 vote has finally passed, we can switch the "stable version tests" over to 1.7 in the near future.